server/lib/generate-alerts/sample-data/mitre.js

```diff
@@ -13,7 +13,7 @@
 // Mitre
 export const arrayMitreRules = [
   {
-    filename: '0015-ossec_rules.xml',
+    filename: '0015-wazuh_rules.xml',
```

Comment:
This file `0015-ossec_rules.xml` exists in my environment with `4.2.0` building from sources (branch `master`) some days ago.
![image](https://user-images.githubusercontent.com/34042064/110359088-5b6a8500-803d-11eb-9afb-a7c6408df63d.png)